metrics/Code_Changes.md

```diff
@@ -0,0 +1,113 @@
+# Code_Changes
+
+Changes to the source code during a certain period.
+
+## Description
+
+These are changes to the source code during a certain period.
+For "change" we consider what developers consider an atomic change to their code.
+In other words, a change is some change to the source code which usually
+is accepted and merged as a whole, and if needed, reverted as a whole too.
+For example, in the case of git, each "change" corresponds to a "commit",
+or, to be more precice, "code change" corresponds to the part of a commit which
+touches files considered as source code.
+
+### Parameters
+
+Mandatory:
+
+* Period of time. Start and finish date of the period. Default: forever.
+
+    Period during which changes are considered.
+    
+* Criteria for source code. Algorithm. Default: all files are source code.
+
+    If we are focused on source code, we need a criteria for deciding
+    whether a file is a part of the source code or not.
+    
+### Aggregators
+
+Usual aggregators are:
+
+* Count. Total number of changes during the period.
+
+## Specific description: git
+
+In the cases of git, a code change is defined as the part of
+a git commit that "touches" a source file.
+That is, for each git commit, only the part related to
+deletions or additions of lines to files that are considered source code
```

Comment:
I seem to remember that we had a definition for _source code_ or discussion about what is considered _source code_ somewhere. How about linking to it to provide clarity for implementers?

Reply:
We have some discussion about this in #72, but I don't think it is conclusive enough... I was planning to work on it on a further version of the implementation, to at least give some clues of how that could be implemented....